charts/nextcloud/values.yaml

```
@@ -666,6 +670,11 @@ cronjob:
     ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
     ##
     resources: {}
+
+    # priorityClassName: priority class for the cron job. Overrides
```

Please use helm-docs annotation
```suggestion
    # -- priorityClassName: priority class for the cron job. Overrides
```